src/openrct2/actions/TileModifyAction.hpp

```diff
@@ -95,11 +96,14 @@ DEFINE_GAME_ACTION(TileModifyAction, GAME_COMMAND_MODIFY_TILE, GameActionResult)
             return MakeResult(GA_ERROR::INVALID_PARAMETERS, STR_LAND_NOT_OWNED_BY_PARK);
         }
         auto res = MakeResult();
+
+        uint8_t speed;
+        uint32_t elementIndex;
```

Comment:
There is no need for this. Use brackets inside your `case` statement, like all other cases already do.